BaseApiBundle/Controller/BaseController.php

```diff
@@ -64,6 +69,11 @@ protected function reverseTransform(Request $request, $id, $type, $event, $event
             $em->persist($mixed);
             $em->flush();
 
+            if(in_array('OpenOrchestra\ModelInterface\Event\EventTrait\EventStatusable', class_uses($eventClass))) {
```

Comment:
It's not the name of a "class" but of a "trait". For me, the line 73 should be done only if the class 'implements' the trait. @itkg-ngilain